rotkehlchen/tests/unit/decoders/test_velodrome.py

```diff
@@ -848,15 +872,17 @@ def test_unstake_lp_token_to_gauge_v2(optimism_accounts, optimism_transaction_de
 
 
 @pytest.mark.vcr()
+@pytest.mark.parametrize('load_global_caches', [[CPT_VELODROME]])
 @pytest.mark.parametrize('optimism_accounts', [['0xf9AEb52bB4eF74E1987dd295E4Df326d41D0d0fF']])
-def test_get_reward_from_gauge_v2(optimism_accounts, optimism_transaction_decoder):
+def test_get_reward_from_gauge_v2(optimism_accounts, optimism_transaction_decoder, load_global_caches):  # noqa: E501
     """Check claiming rewards from a velodrome v2 gauge is properly decoded."""
     evmhash = deserialize_evm_tx_hash('0x9d0eae3c2d2cda853e77a2571a7c702507beb015c08b40406e03baa4b1dde505')  # noqa: E501
     user_address = optimism_accounts[0]
     events, _ = get_decoded_events_of_transaction(
         evm_inquirer=optimism_transaction_decoder.evm_inquirer,
         database=optimism_transaction_decoder.database,
         tx_hash=evmhash,
+        load_global_caches=load_global_caches,
```

Comment:
This is a small but quality of life improvement. If you always pass load_global_caches to this function can't you somehow make it load the value from the fixture automatically? We do